networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/gossip/config/GossipScoringConfig.java

```diff
@@ -163,5 +167,26 @@ public Builder opportunisticGraftThreshold(final Double opportunisticGraftThresh
       this.opportunisticGraftThreshold = opportunisticGraftThreshold;
       return this;
     }
+
+    public Builder directPeers(final List<String> directPeers) {
```

Comment:
I would prefer to have it like below in the Config class:
```java
public Builder directPeers(final DirectPeerManager directPeers)
```
It would allow more flexible (dynamic peer set) and typesafe (`String` is pretty ambiguous) programmatic configuration